### PR TITLE
Add a reusable component Banner

### LINKED ISSUE #32: Create the reusable component Banner
## Goal

- Create a common component `Banner.jsx` used over the app

## Deadline / Task size

- 1 day 

## Steps

- Create `Banner.jsx` in `src/components/common`

## Design

- LP
Image

- Common 
Image

### PR DESCRIPTION
**Pull request by HidemichiShimura**
#### Code changes

- Created the Banner component
    - `Banner.jsx`
    - `BannerContent.jsx`
- Created a scss file for the Banner component
    - `banner.module.scss`
- Added it to the entry point `src/components/index.js`
- Removed `display: flex` from `global.scss`
- Added a layout to LP's main contents with `max-width`

#### Issues affected

- Fixes #32

## Additional Info

I added a layout to LP's main contents with `max-width` in `src/pages/index.js`
let me know if there are misunderstandings or code modifications that need to be applied.

## Screenshots and/or video

- LP
Image

- Common 
Image

#### Basics
- [x] Local QA is complete
- [x] No debugging `console.log()` statements
- [ ] ~~Commented-out code removed~~
- [x] Unused code removed (imports, functions, styles, etc - if it's not used anywhere, it's not in this PR)

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
 import Head from 'next/head'
 
-export default function Home() {
+export default function Home({ children }) {
```
Comment (Soma-dev0808) [inline src/pages/index.js:4]: Can you put layout component in components/common/ ? So that we can import it when we need it.

**Comment by Soma-dev0808** [comment]
I've updated this commit please check it.

File: src/pages/index.js (hunk 1 above)
Comment (HidemichiShimura) [reply src/pages/index.js:4]: Thank you for creating \<PageLayout />
That is a good way to organize the layout, which I did not know!

**Comment by HidemichiShimura** [comment]
Thank you for creating <PageLayout />
That is a good way to organize the layout, which I did not know!

**Pull request merged**